.github/workflows/manifests.yml

```diff
@@ -3,15 +3,15 @@ name: manifests-ci-check
 
 on:
   pull_request:
-    types: [labeled, synchronize]
+    types: [opened, synchronize]
     paths:
       - 'manifests/**/*.yml'
       - '!manifests/templates/**/'
       - 'legacy-manifests/**/*.yml'
```

Comment:
Maybe we can remove this line? Since it's in legacy-manifest folder, we shouldn't modify any of the existing manifests. We will be only move manifests to the legacy folder right?

Reply:
Right but incase we move files from manifest to legacy, the checks will also test for validations. Example, all tags exists and manifests are valid.